src/dotnet/commands/dotnet-new/CSharp_Console/$projectName$.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is a far more complicated template than it needs to be. You should base it of of the already existing MSBuild template and add to it what you need.